godot/Cargo.toml

```diff
@@ -23,6 +23,9 @@ rustfmt = ["godot-core/codegen-rustfmt"]
 lazy-function-tables = ["godot-core/codegen-lazy-fptrs"]
 serde = ["godot-core/serde"]
 
+# default enable in 4.3
+user-docgen = ["godot-macros/docs"]
```

Comment:
What does the comment here mean?

Also, maybe name it `register-docs` since it's closely related to the `register` module.

This feature should also be documented in lib.rs (make sure to rebase on latest `master`, there have been some changes).

Reply:
Its supposed to be a user facing feature, calling it `register` would be confusing

Reply:
`godot::register` is also user-facing and encompasses the whole registration of Rust symbols within Godot.

This feature is about registering not only classes, methods but also their _docs_, so that seems standing to reason.

Reply:
okay then